.github/workflows/run-tests.yml

```diff
@@ -19,9 +19,7 @@ jobs:
           if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
-      - shell: bash
-        run: |
-          ./bootstrap.sh
```

Comment:
Can you bring back those 3 lines please?
If we remove it, our tests in CI will be always against old docker image, so they will not catch recent commit issues.